easy-ui-tsconfig/library.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./dom.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "noEmit": false,
+    "noEmitHelpers": true
+  }
+}
```

Comment:
admittedly i don't know the nuances of all these rules. they were inspired by configuration from other design systems, so i think it can be a good starting point